course_info/static/course_info/js/controllers/DetailsController.js

```diff
@@ -253,7 +253,8 @@
         <input type="text" class="form-control" id="input-course-{{field}}" \
                name="input-course-{{field}}" ng-show="editable" \
                ng-model="formValue" maxlength="{{maxlength}}"/> \
-        <span ng-hide="editable">{{modelValue}}</span> \
+        <span id="span-course-{{field}}"
```

Comment:
Nope, rushed it out for PR before heading out.  I'll add the trailing `\`, fix the leading space, and test before merging.